src/utrition/src/components/imageupload/imageupload.js

```diff
@@ -14,11 +17,81 @@ const ImageUpload = () => {
     upload_type: "image",
   };
 
+  // New header
+  const headers_confirm = {
+    upload_type: "confirm",
+  };
+
+  
   const handleClick = (event) => {
     fileInput.current.click();
   };
 
+  // Doesn't send any response, set food to 'correct', close confirmation
+  function handleCorrectData() {
+    setCorrectFood(true);
+    setShowConfirmationDialog(false);
+  }
+
+  // Sends response to /upload, with new header;
+  // set food to 'incorrect', close confirmation
+  // populate and show error message
```

Comment:
Sends response to backend for a faulty classification, sets the corresponding state variables to reflect a bad classification